service/controller/v7/version_bundle.go

```diff
@@ -12,6 +12,16 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Pin calico-kube-controllers to master.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "kubernetes",
```

Comment:
Why was calico not added here? It should be on 3.6.1 now.

Reply:
Good catch